setup.py

```diff
@@ -19,33 +17,32 @@
 with open('HISTORY.rst') as history_file:
     history = history_file.read()
 
-
-def read(*names, **kwargs):
-    return io.open(
-        join(dirname(__file__), *names),
-        encoding=kwargs.get('encoding', 'utf8')
-    ).read()
-
 requirements = [
-    read('requirements.txt')
+    'python-dateutil==2.6.1',
+    'rasterio==1.0a12',
+    'Click>=6.0',
+    'geojson==2.3.0',
+    'requests==2.18.0',
+    'typing==3.6.2;python_version<"3.5"'
 ]
 
 setup_requirements = [
-    # TODO(notthatbreezy): put setup requirements (distutils extensions, etc.) here
+    # NOQA TODO(notthatbreezy): put setup requirements (distutils extensions, etc.) here
 ]
 
 test_requirements = [
-    # TODO: put package test requirements here
+    # NOQA TODO: put package test requirements here
 ]
 
 setup(
     name='pystac',
-    version='0.1.0',
-    description="Python command line utilities for interacting with and creating STAC compliant files.",
+    version='0.1.1',
```

Comment:
I don't think that we should bump the version in here yet. We should keep that for commits on the release branch.

Reply:
While I working on the deployment portion of the travis config, I deployed a broken `0.1.0` to PyPi. Since PyPi doesn't allow deletions (see pypa/packaging-problems#74), I needed to increment this number so that deploys would work. In the next release, I think we'll need to bump this again.